src/deploy/functions/runtimes/discovery/index.ts

```diff
@@ -7,6 +7,7 @@ import { promisify } from "util";
 import { logger } from "../../../../logger";
 import * as api from "../../.../../../../api";
 import * as backend from "../../backend";
+import * as build from "../../build";
 import * as runtimes from "..";
 import * as v1alpha1 from "./v1alpha1";
 import { FirebaseError } from "../../../../error";
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"endLine":17,"prLink":"https://app.graphite.dev/github/pr/firebase/firebase-tools/4632"}} -->
<i class='graphite__hidden'>[Re: line 17]<br/></i>

* is `yamlToBackend` still needed<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/4632?utm_source=unchanged-line-comment">Graphite</a>.</span>

Reply:
It's not needed anymore. My plan was to rip it (and backendFromV1Alpha1) out only after this release, when we were sure we wouldn't need to go backwards.